src/main/java/com/volmit/iris/engine/object/IrisObject.java

```diff
@@ -865,7 +852,12 @@ public int place(int x, int yv, int z, IObjectPlacer oplacer, IrisObjectPlacemen
             }
         }
 
-        return y;
+        if (config.getRawCommands().isNotEmpty()) {
+            Location l = new Location(rdata.getEngine().getWorld().realWorld(), x, y, z);
+            for (IrisCommand rawCommand : config.getRawCommands()) {
```

Comment:
No, objects cannot support this do not do this unless you want to put "prepared" commands in the mantle. Honestly just remove all changes in the object classes